### PR TITLE
Krystophv/unflex

### PR DESCRIPTION
**Pull request by krystophv**
An effort to remove the global div/span flex styles and apply the flex layout options only to the relevant pieces of the application.

### CODE REVIEW

**Comment by krystophv** [comment]
@jebeck could you take a look at this one. I'd like to get this in before migration of everything to CSS modules.

File: styles/components/App.less [hunk 1]
```diff
@@ -15,12 +15,16 @@
 
 .App {
   height: 100%;
+  display: flex;
+  flex-direction: column;
```
Comment (jebeck) [inline styles/components/App.less]: Seems like a mixin for the combo of these two props would be good, but maybe not now but rather as part of the CSS modules port/single resp refactor?

File: styles/components/VersionCheck.less [hunk 2]
```diff
@@ -56,7 +56,7 @@
       // (effectively block)
       // TODO: fix that at the source
       span {
-        display: inline;
+        // display: inline;
```
Comment (jebeck) [inline styles/components/VersionCheck.less]: Remove the commented out line plz?

File: styles/core/dropdowns.less [hunk 3]
```diff
@@ -3,7 +3,7 @@
 // because we're setting display: flex on every div
 // and it turns out that interferes with text-overflow :(
 .Select-placeholder {
-  display: block;
+  //display: block;
```
Comment (jebeck) [inline styles/core/dropdowns.less]: Ditto.

File: styles/core/scaffolding.less [hunk 4]
```diff
@@ -17,7 +17,7 @@
 // ====================================
 
 div, span {
-  .box-flex();
+  //.box-flex();
```
Comment (jebeck) [inline styles/core/scaffolding.less]: And ditto. :wink:

File: styles/components/App.less (hunk 1 above)
Comment (krystophv) [inline styles/components/App.less]: Yeah, likely as part of a layout.css that we can compose from.

**Comment by jebeck** [comment]
:+1: Also gave it a quick visual spot check locally.

**Comment by krystophv** [comment]
https://trello.com/c/mg3NudYP/23-tech-debt-uploader-remove-display-flex-global-style-and-fix-what-breaks

**Pull request merged**